### DISCUSSION TITLE
There is a problem running page behind reverse proxy

### DISCUSSION THREAD

**Discussion by arturslogins** [bagisto/bagisto#10338] [Q&A] [answered]
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

latest

### Issue Description

I configured pages behind reverse proxy. 
Also in .env file added my domain 

But im getting errors on https 

Also i trayed to force https  in AppServiceProvider  boot function,  but it brokes upload in admi side. 
And i can't figure out, where is problem 

Request URL:
https://test.MY_DOMAIN.com/admin/configuration/general/APP_URL=https://test.MY_DOMAIN.com/storage/configuration/dbH77eoQuHFoBxZr5B811cZ6a0ZxIP4s5kahuQOk.png
Request Method:
GET
Status Code:
500 Internal Server Error

test.profoutlet.lv/:1 Mixed Content: The page at 'https://test.MY_DOMAIN.com/' was loaded over HTTPS, but requested an insecure favicon 'http://test.MY_DOMAIN.com/themes/shop/default/build/assets/favicon-Df9chQdB.ico'. This request has been blocked; the content must be served over HTTPS.

Im running all in docker containers 

### Preconditions

Im running all in docker containers   using https://github.com/bagisto/bagisto-docker 
And as a loadbalancer im using Traefik 

services:
    profoutlet:
        build:
            context: ./profoutlet
            dockerfile: Dockerfile
        image: profoutlet
        labels:
           - "traefik.enable=true"
           - "traefik.http.routers.profoutlet.rule=Host(`test.MY_DOMAIN.com`)"
           - "traefik.http.routers.profoutlet.tls=true"
           - "traefik.http.routers.profoutlet.tls.certresolver=letsencrypt"
           - "traefik.http.services.profoutlet.loadbalancer.server.port=80"
           - "traefik.http.routers.profoutlet.service=profoutlet"
           - "traefik.http.middlewares.profoutlet.headers.customrequestheaders.X-Forwarded-Proto=https"
        volumes:
            - './profoutlet:/srv/www'
        networks:
            - prof

### Expected Result

Working using https

### Actual Result

Not working with https

**Comment by devansh-webkul** [comment, marked as answer]
Kindly try enforcing HTTPS in the service provider or configure the trust proxies.

**Comment by eduartua** [comment]
> ### Terms
> * [x]  Before you openning this issue, i have checked if the issue has already been reported.
> 
> ### Bagisto Version(s) affected
> latest
> 
> ### Issue Description
> I configured pages behind reverse proxy. Also in .env file added my domain
> 
> But im getting errors on https
> 
> Also i trayed to force https in AppServiceProvider boot function, but it brokes upload in admi side. And i can't figure out, where is problem
> 
> Request URL: https://test.MY_DOMAIN.com/admin/configuration/general/APP_URL=https://test.MY_DOMAIN.com/storage/configuration/dbH77eoQuHFoBxZr5B811cZ6a0ZxIP4s5kahuQOk.png Request Method: GET Status Code: 500 Internal Server Error
> 
> test.profoutlet.lv/:1 Mixed Content: The page at 'https://test.MY_DOMAIN.com/' was loaded over HTTPS, but requested an insecure favicon 'http://test.MY_DOMAIN.com/themes/shop/default/build/assets/favicon-Df9chQdB.ico'. This request has been blocked; the content must be served over HTTPS.
> 
> Im running all in docker containers
> 
> ### Preconditions
> Im running all in docker containers using https://github.com/bagisto/bagisto-docker And as a loadbalancer im using Traefik
> 
> services: profoutlet: build: context: ./profoutlet dockerfile: Dockerfile image: profoutlet labels: - "traefik.enable=true" - "traefik.http.routers.profoutlet.rule=Host(`test.MY_DOMAIN.com`)" - "traefik.http.routers.profoutlet.tls=true" - "traefik.http.routers.profoutlet.tls.certresolver=letsencrypt" - "traefik.http.services.profoutlet.loadbalancer.server.port=80" - "traefik.http.routers.profoutlet.service=profoutlet" - "traefik.http.middlewares.profoutlet.headers.customrequestheaders.X-Forwarded-Proto=https" volumes: - './profoutlet:/srv/www' networks: - prof
> 
> ### Steps to reproduce
> 1. Go to '...'
> 2. Click on '....'
> 3. Scroll down to '....'
> 4. See error
> 
> ### Expected Result
> Working using https
> 
> ### Actual Result
> Not working with https
> 
> ### Screenshots
> _No response_
> 
> ### Additional context
> _No response_

@arturslogins  Did you ever fixed this by not touching the service provider?
